start/scaling-up/preparing-to-scale-up.md

```diff
@@ -1,92 +1,142 @@
+Below are some suggestions for making resource requests for initial test 
+jobs. **As always, reviewing the HTCondor `log` file from past jobs is 
+a great way to learn about the resource needs of your jobs.**
+
+- Save the HTCondor `log` files from your jobs. HTCondor will report 
+the memory, disk, and cpu usage of your jobs to this file.
+
+- Start by requesting a single cpu. With single cpu jobs you will see 
+your jobs start sooner. Ultimately you will be able to achieve 
+greater throughtput with single cpus jobs compared to jobs that request 
+and use multiple cpus. 
+	- **Keep in mind, requesting more cpus for a job 
+does not automatically mean that your jobs will use more cpus.**
+
+	- There is limited support for multicore work in OSG. To 
+learn more, see our guide on 
+[Multicore Jobs](https://support.opensciencegrid.org/support/solutions/articles/5000653862)
+
+- Before submitting a job always look at the size of your input 
+files. At a minimum you need to request enough disk to support all 
+of the input files your job will need, but don't forget about 
+job output. Your submit file should also request enough disk to 
+support the results that are produced by the job.
+	- If many of your input and output files are compressed 
+(i.e. zipped or tarballs) you will need to factor that into your 
+estimates for disk usage as these files will be uncompressed during 
+job execution.
+
+	- For your initial tests it is OK to request more disk than 
+your job may need so that the test completes successfully. **The key 
+is to adjust disk requests for subsequent jobs based on the results 
+of these test jobs.**
+
+- Estimating memory usage can sometimes be tricky. If you've performed the 
+same or similar work on another computer, consider using the amount of 
+memory (i.e. RAM) from that computer as a starting point. For instance, 
+most laptop computers these days will have 8 or 16 GB of memory.
+	- For your initial tests it is OK to request more memory than 
+your job may need so that the test completes successfully. **The key 
+is to adjust memory requests for subsequent jobs based on the results 
+of these test jobs.**
+
+	- If you find that memory usage will vary greatly across a 
+batch of jobs, we can assist you with creating dynamic memory requests 
+in your submit files.
+
+# Submit Multiple Jobs Using A Single Submit File
+
+Once you have a single test job that compeltes successfully, the next 
+step is to submit a small batch of test jobs (e.g. 5 or 10 jobs) 
+[**using a single submit file**](https://support.opensciencegrid.org/support/solutions/articles/12000073165). Use this small-scale multi-job submission test to ensure that all jobs 
+complete successfully, produce the desired output, and do not 
+conflict with each other when submitted together. Once 
+you are confident that the jobs will complete as desired, then scale up to submitting 
+the entire set of jobs.
+

```

Comment:
Do we want to add a note about using `condor_history` to see how much memory/disk jobs used? The history file turns over pretty quickly, but it's much easier than looking through log files. 

I think you could add (yet another) note here about checking the log files.

Reply:
i have added note about `condor_history` as well as give an example of using `grep` to pull detail from the log file in commit  b69ebd8